dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -786,21 +786,21 @@ BEGIN
     IDS_ABOUT_VERSION_STRING "Versiune %S (%S)"
 
     /* context menus */
-    IDS_VIEW_LARGE "D&ale"
+    IDS_VIEW_LARGE "&Dale"
     IDS_VIEW_SMALL "&Pictograme"
```

Comment:
You changed that to "&Miniaturi" in the top of the file. So you are obliged to use the same string here.